tests/system/providers/dbt/cloud/example_dbt_cloud.py

```diff
@@ -77,6 +77,12 @@
     )
     # [END howto_operator_dbt_cloud_run_job_sensor]
 
+    # [START howto_operator_dbt_cloud_run_job_sensor_defered]
+    job_run_sensor_defered = DbtCloudJobRunSensor(
+        task_id="job_run_sensor_defered", run_id=trigger_job_run2.output, timeout=20
+    )
+    # [END howto_operator_dbt_cloud_run_job_sensor_defered]
+
     # [START howto_operator_dbt_cloud_run_job_async_sensor]
     job_run_async_sensor = DbtCloudJobRunAsyncSensor(
         task_id="job_run_async_sensor", run_id=trigger_job_run2.output, timeout=20
```

Comment:
This sensor should be removed from the system test now that it's deprecated IMO.

Reply:
I think the test will fail if we remove it as not all the sensors are covered by at these one example

Reply:
Ah yes, you're right.